code/datums/diseases/advance/symptoms/radiation.dm

```diff
@@ -0,0 +1,51 @@
+/datum/symptom/radiation
+	name = "Iraddiant Cells"
+	desc = "Causes the cells in the host's body to give off harmful radiation."
```

Comment:
```suggestion
	desc = "Causes the cells in the host's body to give off ionising radiation."
```

Reply:
eh i guess my thought process was to add more diversity to level 8